rollup.go

```diff
@@ -0,0 +1,44 @@
+package gosnowth
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"path"
+	"strings"
+	"time"
+)
+
+type RollupValues struct {
+	Timestamp int64
+	Value     float64
+}
+
+func (rv *RollupValues) UnmarshalJSON(b []byte) error {
+	tt := []interface{}{&rv.Timestamp, &rv.Value}
+	json.Unmarshal(b, &tt)
+	if len(tt) != 2 { // error not enough fields
```

Comment:
can tt come back as greater than 2?  generally speaking in code I prefer </<=/>/>= always to make sure something doesn't get by when its wrong in an unexpected way...

Reply:
yeah, its possible for that array to return more, and I guess it makes sense to be permissive.